src/library/scala/collection/Seq.scala

```diff
@@ -266,15 +266,15 @@ trait SeqOps[+A, +CC[_], +C] extends Any
     *  @return `true` if this $coll has `that` as a suffix, `false` otherwise.
     */
   def endsWith[B >: A](that: Iterable[B]): Boolean = {
-    val i = iterator.drop(length - that.size)
-    val j = that.iterator
-    while (i.hasNext && j.hasNext)
-      if (i.next() != j.next())
-        return false
+      val i = iterator.drop(length - that.size)
+      val j = that.iterator
 
-    !j.hasNext
-  }
+      while (i.hasNext && j.hasNext)
+        if (i.next() != j.next())
+          return false
 
+      !j.hasNext
+  }
```

Comment:
I would get rid of the changes to this file, since they're just changes to indentation.

Reply:
I left this out when I squashed, thanks for pointing this out.